src/components/withWindowDimensions.js

```diff
@@ -0,0 +1,60 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import {Dimensions} from 'react-native';
+
+export const windowDimensionsPropTypes = {
```

Comment:
I think we discussed in the changes to the JS style guide that we shouldn't use inline exports. If I am remembering correctly, then I think this just needs to have `export windowDimensionsPropTypes;` moved to the bottom of the file.

Reply:
Ah cool I interpreted this as "if a file has multiple named exports they should be exported at the bottom of the file together". Didn't realize we wanted to prevent exports like this when we are mixing named/default.